modal/volume.py

```diff
@@ -693,8 +693,12 @@ async def read_file(self, path: str) -> AsyncIterator[bytes]:
         except modal.exception.NotFoundError as exc:
             raise FileNotFoundError(exc.args[0])
 
+        @retry(n_attempts=5, base_delay=0.1, timeout=None)
         async def read_block(block_url: str) -> bytes:
             async with ClientSessionRegistry.get_session().get(block_url) as get_response:
+                if get_response.status in [503, 429]:
+                    logger.warning("Received SlowDown signal, sleeping for 1 second before retrying.")
```

Comment:
Is this expected to happen during normal operation? Not very actionable for users

Reply:
I'm just cargo-culting this tbh:

https://github.com/modal-labs/modal-client/blob/e0757d94cef8d3c40e54302eda1450cd465471ff/modal/_utils/blob_utils.py#L87-L89

Happy to remove both the `warning` and the extra `sleep`.